opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -950,7 +950,8 @@ public void prepareIndexer(RuntimeEnvironment env,
             long start = System.currentTimeMillis();
             env.setRepositories(env.getSourceRootPath());
             long time = (System.currentTimeMillis() - start) / 1000;
-            LOGGER.log(Level.INFO, "Done scanning for repositories ({0}s)", time);
+            LOGGER.log(Level.INFO, "Done scanning for repositories, found {1} repositories ({0}s)",
+                    new Object[]{time, env.getRepositories().size()});
```

Comment:
I know that this is just a matter of preference, but can you keep the objects in the order they appear in the string?